src/nxdoc/nuxeo-server/platform-services/automation/automation-scripting.md

```diff
@@ -320,40 +320,31 @@ If one of your Automation chain or operation contains dashes (`-`) in their name
 
 {{/callout}}
 
-The following code example is bound to a simple action button, renames all selected documents with the value "test" as `dc:title` or displays a warning JSF message (depending on the number of the selected documents).
+The following code example is bound to a simple event (`Document created`), to apply property value inheritance from the parent document, with some logging information messages.
 
 ```js
 function run(input, params) {
-    var docs = Seam.GetSelectedDocuments(input, {});
-    /* Description: Fetch the documents selected in the current folder listing */
-    if (docs.length > 3) {
-        var index;
-        for (index = 0; index < docs.length; ++index) {
-            Document.SetProperty(input, {
-                /* required: true - type: string */
-                "xpath": "dc:title",
-                /* required: false - type: boolean */
-                "save": true,
-                /* required: false - type: serializable */
-                "value": "test"
-            });
-        }
-        WebUI.Refresh(input, {});
-    } else {
-        WebUI.AddMessage(input, {
-            /* required: true - type: string */
-            "message": "DISPLAY IT",
-            /* required: true - type: string */
-            "severity": "WARN"
+
+ Console.log("Starting property inheritance");
+
+  var parent = Document.GetParent(input, {
+      /*required:false - type: string*/
+      'type': 'product'
+      }
+  );
+
+  if (parent) {
+        input = Document.CopySchema(input, {
+        /*required:true - type: string*/
```

Comment:
this part should be indented regarding the previous line

Reply:
👍